README.md

```diff
@@ -485,7 +486,10 @@ will output a image section
 <figure class="image embedded">
   <div class="img">
     <img src="http://example.com/lovely-landscape.jpg" alt="Some alt text">
-    <figcaption>An optional caption</figcaption>
+    <figcaption>
+      <p>An optional caption</p>
+      <p>An optional credit</p>
```

Comment:
Perhaps you want to prefix this with "Image credit:"

Reply:
Updated and fixed-up.